src/org/opendatakit/briefcase/ui/export/components/FormsTableViewModel.java

```diff
@@ -36,6 +36,7 @@
 import javax.swing.table.AbstractTableModel;
 import org.opendatakit.briefcase.export.ExportConfiguration;
 import org.opendatakit.briefcase.export.ExportForms;
+import org.opendatakit.briefcase.model.BriefcasePreferences;
```

Comment:
This applies down below, where there wasn’t a change in this PR:
>View this form's status history
Can we improve on this? It’s more of an activity log or some such, no? Also, the message appears in two places in the code, and it has a straight quote (') instead of ’.